scipy/stats/tests/test_distributions.py

```diff
@@ -97,6 +97,67 @@ def test_all_distributions():
         yield check_distribution, dist, args, alpha
 
 
+def check_rvs_broadcast(distfunc, distname, args, loc, scale, shape):
+    sample = distfunc.rvs(*args, loc=loc, scale=scale)
+    assert_equal(sample.shape, shape, "%s: rvs failed to broadcast" % distname)
+
+
+def test_all_rvs_broadcast_shape():
+    # This test only checks the *shape* of the value returned by rvs().
+    for dist in dists:
```

Comment:
Note that `dists` is a hard coded list which does not include all distributions IIRC. (I actually wonder why this is so, but that's off-topic)